static/js/components/CommentAttachmentPreview.jsx

```diff
@@ -251,7 +259,7 @@ const CommentAttachmentPreview = ({
                 // preview with no way to edit it without losing resolution due
                 // to hard-coded in-line styling, so use the FilePreviewer
                 // component for PDF
-                <FilePreviewer file={{ url }} hideControls />
+                <FilePreviewer file={{ url: previewUrl }} hideControls />
```

Comment:
Doesn't this override the expected behavior when the attachment is a PDF?

Reply:
Good point. fixed.